### ISSUE TITLE
Remove external dependency on Cloudflare's cdnjs

### ISSUE THREAD

**Issue by sstangl** [sstangl/openpowerlifting#279]
A bunch of the HTML pages now link to https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css.

Just now, for me, resolution of this address caused a ~1.5 minute delay in initial site rendering.

It would be good to host all dependencies ourselves. Additionally, that would let us minify them further based on our actual usage.

**Comment by emilyytlin** [comment]
I don't think it takes an extra 1.5 minutes because it needs to go to cloudflare. For example, the data and contact pages don't take that long to load. I chose to use a cdn because they're designed to serve files quickly.
